pkgs/by-name/ca/cassette-framework/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
```

Comment:
```suggestion
```

not used anywhere in this file

Reply:
You're right, my mistake, I didn't look carefully before committing.
I'll rebase soon to have only one commit.